PocketSql.Tests/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this app.config needed for was it automatically created by visual studio?

Reply:
fixed.